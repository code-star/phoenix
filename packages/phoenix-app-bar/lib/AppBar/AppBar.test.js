import * as renderer from 'react-test-renderer'
import {
  simpleAppBar,
  denseAppBar,
} from './AppBar.render'

describe('<AppBar />', () => {
  describe('Snaphot', () => {
    test('must match simpleAppBar', () => {
      expect(renderer.create(simpleAppBar).toJSON()).toMatchSnapshot()
    })
    test('must match denseAppBar', () => {
      expect(renderer.create(denseAppBar).toJSON()).toMatchSnapshot()
    })
  })
})
