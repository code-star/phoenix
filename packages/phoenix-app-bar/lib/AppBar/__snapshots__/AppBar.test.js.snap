// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> Snaphot must match denseAppBar 1`] = `
<div
  className="DenseAppBar-root-77"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorPrimary-9"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-dense-41 MuiToolbar-gutters-39"
    >
      <button
        aria-label="Menu"
        className="MuiButtonBase-root-85 MuiIconButton-root-79 MuiIconButton-colorInherit-80 DenseAppBar-menuButton-78"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-84"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-88"
            focusable="false"
            onClick={[Function]}
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-97"
        />
      </button>
      <h6
        className="MuiTypography-root-42 MuiTypography-h6-59 MuiTypography-colorInherit-71"
      >
        Dense App Bar
      </h6>
    </div>
  </header>
</div>
`;

exports[`<AppBar /> Snaphot must match simpleAppBar 1`] = `
<div
  className="SimpleAppBar-root-1"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorDefault-8"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
    >
      <h6
        className="MuiTypography-root-42 MuiTypography-h6-59 MuiTypography-colorInherit-71"
      >
        Dense App Bar
      </h6>
    </div>
  </header>
</div>
`;
